src/Analyzers/CSharp/Analyzers/UseSimpleUsingStatement/UseSimpleUsingStatementDiagnosticAnalyzer.cs

```diff
@@ -98,15 +100,22 @@ private void AnalyzeSyntax(SyntaxNodeAnalysisContext context)
             }
 
             // Verify that changing this using-statement into a using-declaration will not change semantics.
-            if (!PreservesSemantics(semanticModel, parentBlock, outermostUsing, innermostUsing, cancellationToken))
+            if (!PreservesSemantics(semanticModel, parent, outermostUsing, innermostUsing, cancellationToken))
                 return;
 
             // Converting a using-statement to a using-variable-declaration will cause the using's variables to now be
             // pushed up to the parent block's scope. This is also true for any local variables in the innermost using's
             // block. These may then collide with other variables in the block, causing an error.  Check for that and
             // bail if this happens.
+            var variableScopeNode = parent switch
+            {
+                BlockSyntax block => block,
+                GlobalStatementSyntax global => global.Parent!,
```

Comment:
```suggestion
                GlobalStatementSyntax globalStatement => globalStatement.Parent!,
```
I think this name reads better